bftengine/src/bcstatetransfer/RangeValidationTree.cpp

```diff
@@ -150,8 +150,8 @@ const shared_ptr<char[]> RVBNode::computeNodeInitialValue(NodeInfo& node_info, c
   ConcordAssertGT(node_info.id(), 0);
   DigestGenerator digest_generator;
 
-  digest_generator.update(reinterpret_cast<const char*>(&node_info.id_data_), sizeof(node_info.id_data_));
-  digest_generator.update(data, data_size);
+  digest_generator.updateDigest(reinterpret_cast<const char*>(&node_info.id_data_), sizeof(node_info.id_data_));
```

Comment:
As the class name is DigestGenerator, update is a better name here.

Reply:
Done.